src/pages/events/detail.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from "@emotion/styled";
+import { NextPage } from "next";
+import EventDetail from "../../components/EventDetail";
+import EventsList from "../../components/EventsList";
+
+const EventDetailPage: NextPage = () => {
+  return (
+    <Container>
+      <EventsList />
+      <EventDetail />
+    </Container>
+  );
+};
+
+const Container = styled.section`
+  padding-top: 72px;
+  padding-left: 96px;
+`;
```

Comment:
p4: 저는 개인적으로는 이렇게 명시해줘도 나쁘지 않다고 생각합니다. 지윤님의 의견으로는 패딩 속성이 2개 이상이니 한번에 합쳐도 좋을 것 같아요. 
```suggestion
  padding:72px 0 0 96px;
`;
```

Reply:
반영할게요!